src/littlefs/context.py

```diff
@@ -74,3 +75,89 @@ def sync(self, cfg: 'LFSConfig') -> int:
             Filesystem configuration object
         """
         return 0
+
+
+
+# if the user has win32file installed then define this class 
+try:  
+    import win32file  
+except ImportError:
+    win32file = None
+else:
```

Comment:
so sorry, with this new implementation (checking `win32file` in `__init__`), then we no longer need this `else`:

```
try:  
     import win32file  
 except ImportError:
     win32file = None
 
class UserContextWinDisk(UserContext):
    ...
```

Otherwise right now we get ImportErrors if `win32file` is not available.